src/sinks/elasticsearch/common.rs

```diff
@@ -140,6 +141,24 @@ impl ElasticsearchCommon {
         })
     }
 
+    /// Parses endpoints.
+    /// Not empty.
+    pub async fn parse_endpoints(config: &ElasticsearchConfig) -> crate::Result<Vec<Self>> {
```

Comment:
:thinking: the function name here seems like it could be improved, but one isn't coming to mind. Its more like the config is being parsed and the endpoints from the config are being used to create commons. `create_commons_from_endpoints()` ? Ideas welcome :)

Reply:
Maybe `parse_many`? Then the helper function for 
```
 let common = ElasticsearchCommon::parse_endpoints(&config)
        .await
        .expect("Config error");
        .expect("Config error")
        .remove(0);
```
could be named `parse_single`.

Reply:
:+1:  I like that